Runtime/Core/CinemachineComponentBase.cs

```diff
@@ -248,5 +248,8 @@ public CameraState VcamState
         /// </summary>
         /// <returns>Highest damping setting in this component</returns>
         public virtual float GetMaxDampTime() { return 0; }
+
+        /// <summary>Components that require input should implement this and return true.</summary>
+        public virtual bool RequiresInput => false;
```

Comment:
The name of this method makes sense if we're already in the context of thinking about mouse input etc, but to someone just looking at the class as a whole, "Input" is vague.  It could mean anything.  Please choose a name that's more specific, maybe UserInput?